numpy/ma/core.py

```diff
@@ -2988,7 +3016,7 @@ def __array_finalize__(self, obj):
             except ValueError:
                 self._mask = nomask
             except (TypeError, AttributeError):
-                # When _mask.shape is not writable (because it's a void)
+                # When _mask.shape is not writeable (because it's a void)
```

Comment:
You could say it's not particular... writeable.

In numpy we should write it `writeable`, because `arr.flags.writeable` is part of the public API.